charts/karpenter/values.yaml

```diff
@@ -27,6 +27,10 @@ serviceMonitor:
   additionalLabels: {}
   # -- Endpoint configuration for the ServiceMonitor.
   endpointConfig: {}
+rbac:
+  # -- Specifies if Aggregated ClusterRoles should be created.
+  # If set to true, the 'admin' ClusterRole will be able to manage Karpenter provisioners
+  createAggregateClusterRoles: false
```

Comment:
let's minimize branching and just include it by default. If someone wants to disable it, we can do so then.

Reply:
Done 👍